### PR TITLE
Added valid STEPs info to clean command

### PR DESCRIPTION
**Pull request by dplanella**
Added information of the valid STEP values to pass to `snapcraft clean -s`. The extra info will be shown on the output of `snapcraft clean --help`.

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: snapcraft/commands/clean.py [hunk 1]
```diff
@@ -26,7 +26,8 @@
 Options:
   -h --help             show this help message and exit.
   -s STEP --step=STEP   only clean the specified step and those that depend
-                        upon it.
+                        upon it. STEP can be one of: pull, build, stage, strip,
+                        snap. See 'snapcraft help plugins' to learn more.
```
Comment (kyrofa) [inline snapcraft/commands/clean.py]: Ah, good addition, thank you! However, these steps are per-part, and `snap` isn't a per-part step, so it's just pull, build, stage, or strip.

**Comment by kyrofa** [comment]
ok to test

**Comment by dplanella** [comment]
Thanks Kyle, updated as per the comment.

**Comment by kyrofa** [comment]
@dplanella looks good. If you could please rebase onto master and squash into a single commit, we'll get this in :smiley: .

**Comment by kyrofa** [comment]
Fixed in #423.

**Pull request closed without merging**